docs/_data/repoStatus.yaml

```diff
@@ -231,7 +231,7 @@
       status: N/A
     - name: Documentation
       status: Ready
-- tagName: rh-navigation-primary      
+- tagName: rh-navigation
```

Comment:
I thought we had discussed at one point that the `rh-navigation` would still have the `-primary` on it?  maybe I'm mistaken. @markcaron

Reply:
@zeroedin I believe that was the decision, yeah.